README.md

```diff
@@ -1,94 +1,46 @@
 # A Reference Process for OSS Projects on GitHub
 
-This is an attempt to document similarities in in a number of our OSS projects hosted on GitHub. 
+This is an attempt to document practices common to a number of OSS projects we have on GitHub. 
```

Comment:
@pyokagan 

>The name of the organization reflects its purpose. For me, the name "oss-generic" leads me to wonder (and worry) about the scope of the project.
Who/What is this project catering to? Is it catering to TEAMMATES, SE-EDU, PowerPointLabs? Or is it catering to the OSS community at large?

This is a valid point. My response is, _the project currently caters for a selected set of projects but has the ambition to cater for a wider community, if the wider community accepts it_. 

1. In the more likely case that no one else other than us take note of this repo, there is no problem. 
2. In the case it catches on and we are forced to cater for the wider community's demands at the expense of our own projects, that's a _happy problem_ isn't it? It would be great achievement for all of us involved if the wider community adopts it. So shouldn't we keep the door open for that possibility (however small it is) rather than close it for sure? At the moment, keeping that door open doesn't cost us anything. 

But I kind of feel that it's a bit arrogant(?) to name it OSS-Generic. As I mentioned above, I'm open for a better name that doesn't make sweeping claims but also not NUS specific. The problem is, NUS is the only common thing among our projects. I can't very well name it `Process for projects done by Damith's students` :-D